titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/DefaultV3JobOperations.java

```diff
@@ -243,10 +243,9 @@ public void shutdown() {
                     if (reservationFailure.isPresent()) {
                         return Observable.error(JobManagerException.jobCreateLimited(reservationFailure.get()));
                     }
-                    Optional<String> limited = jobSubmitLimiter.checkIfAllowed(jobDescriptorWithCallerId);
-                    if (limited.isPresent()) {
-                        jobSubmitLimiter.releaseId(jobDescriptorWithCallerId);
```

Comment:
Is it safe to remove this line?

Reply:
Good catch. Didn't mean to remove it from here